knowledge-base/kb-gridview-filteringdisctinctvalues.md

```diff
@@ -0,0 +1,48 @@
+---
+title: RadGridView Column Filtering Enum Values
+description: "How to filter Enum values in RadGridView column."
+type: how-to
+page_title: Filtering distinct values in RadGridView column for WPF.
```

Comment:
Each word in the "title" and "page_title" properties should begin with an uppercase letter.